source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,140 @@ weight: 70
 
 # Making payments
 
-When you make a payment, you will need to supply a ``reference``. If possible,
+When you make a payment, you will need to supply a `reference`. If possible,
 this should be a unique identifier for the payment. If your organisation
-already has an existing identifier for payments, you can use it here.
+already has an existing identifier for payments, you can use it as this
+`reference`.
 
-You will also need to supply a ``return_url``, a URL hosted by your service
-for the user to return to after they have completed payment on GOV.UK Pay. See
-the section below on **Choosing the return URL** for more information.
+You will also need to supply a
+[`return_url`](/#choosing-the-return-url-and-matching-user-to-payment). This
+is a URL that your service hosts for the user to return to, after they have
+completed a payment using GOV.UK Pay.
 
-You will need to store the URL from the Location header or in the ``self``
-section of ``_links`` in the JSON body (the same URL is shown in both places).
-This URL contains the GOV.UK Pay ``payment_id`` which identifies the
-payment. An authenticated GET request to the URL will return information about
-the payment and its status.
+You will need to store the URL from the `Location` response header, or from
+the `self` section of `_links` in the JSON body.  This URL contains the GOV.UK
+Pay `payment_id`, which identifies the payment. You can make a GET request to
```

Comment:
The way this is worded (I haven't changed this much from the original yet) seems to suggest that `payment_id` and `reference` are the same, or at least serve the same function. Is there an obvious thing to point to to distinguish them?

Reply:
There’s the fact that the `payment_id` is automatically generated by us whereas the `reference` is generated by the service. Can you do anything with that?